content/applications/sales/point_of_sale/restaurant/fiscal_position.rst

```diff
@@ -1,43 +1,51 @@
-============================
-Tax rates (fiscal positions)
-============================
+=================================
+Flexible taxes (fiscal positions)
+=================================
 
-In Odoo, *Fiscal Positions* let you apply different taxes based on
-the customer location. In a *Point of Sale*, such as a restaurant, it can
-be used to apply different taxes depending if the customer eats in or
-takes away.
+When running a business, you may need to apply different taxes and record transactions on various
+accounts based on the location and type of business of your customers and providers.
 
-Set up fiscal positions for PoS
-===============================
+The **fiscal positions** feature enables you to establish rules that automatically adjust
+the taxes and accounts used for each transaction.
 
-To enable this feature, go to :menuselection:`Point of Sale --> Configuration --> Point of Sale`
-and check *Fiscal Position per Order*. Now, you can choose the fiscal positions
-you want for your *PoS*.
+.. seealso::
+   - :doc:`../../../finance/accounting/taxation/taxes/fiscal_positions`
+   - :doc:`../../../finance/accounting/taxation/taxes/taxes`
 
-.. image:: fiscal_position/fiscal_position_01.png
-    :align: center
+Configuration
+=============
 
-.. note::
-   You need to create your fiscal positions before using this feature.
+To enable the feature, go to :menuselection:`Point of Sale --> Configuration --> Settings`, scroll
+down to the :guilabel:`Accounting` section, and enable :guilabel:`Flexible Taxes`.
+
+Then, set a default fiscal position that should be applied to all sales in POS in the
```

Comment:
"to all sales in POS" sounds like this setting is applied to the whole app (which would be "Point of Sale" instead, then)

It isn't clear here that this action is applied to the current pos. I know it's the case for all settings, but it wouldn't hurt to add "select the pos" at the beginning.